### DISCUSSION TITLE
How to merge vcfs of different samples?

### DISCUSSION THREAD

**Discussion by zyxNo1** [etal/cnvkit#1014] [General]
I apologize for the inconvenience, but I am confused when follow the CNVkit user guide since it seems to only provide instructions for CNV calling for single germline sample or paired samples for somatic calling. Thus, when attempting to call germline CNVs for cohorts, I have been following the germline mode, calling one sample at a time. Consequently, I have ended up with numerous VCF files, each containing data from only one sample.
My question is: 
Are there any scripts available in CNVkit for cohort calling or merging the output VCFs that I may have overlooked? If not, could you kindly recommend a method to address this issue?

**Comment by etal** [comment]
Probably GISTIC2.
